schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -1368,42 +1405,44 @@
     "Spec.Distribution.Modules.Monitoring.Mimir": {
       "type": "object",
       "additionalProperties": false,
+      "description": "Configuration for the Mimir package.",
       "properties": {
         "retentionTime": {
           "type": "string",
-          "description": "The retention time for the mimir pods"
+          "description": "The retention time for the logs stored in Mimir."
```

Comment:
Can we also add an example of the values here?

Reply:
Done, added default value and accepted format.

Reply:
Nice!